docker-compose.yml

```
@@ -16,11 +16,10 @@ services:
     ports:
       - 3001:3001
     environment:
```

```suggestion
    user: node
    environment:
```
@cleytonoliveira Adicionando essa configuração o usuário do container passa a não ser mais `root`, o que resolveu aqui o problema de rodar os comandos do `sequelize` usando o script do `npm`.